server/src/services/duplicate.service.ts

```diff
@@ -38,9 +38,11 @@ export class DuplicateService {
   }
 
   async getDuplicates(auth: AuthDto): Promise<DuplicateResponseDto[]> {
-    const res = await this.assetRepository.getDuplicates({ userIds: [auth.user.id] });
-
-    return mapDuplicateResponse(res.map((a) => mapAsset(a, { auth, withStack: true })));
+    const duplicates = await this.assetRepository.getDuplicates(auth.user.id);
+    return duplicates.map(({ duplicateId, assets }) => ({
+      duplicateId,
+      assets: assets.map((asset) => mapAsset(asset, { auth })),
```

Comment:
I don't think it makes sense for how duplicate detection works right now. It won't detect the whole stack as being a duplicate, just particular members of that stack if they're similar enough. The primary asset could be a duplicate, but others might not be and vice versa. This could be addressed in a separate PR, though.

Reply:
I'm not against removing it, but I think it does make sense for those types of decisions to be transparently made, which most likely means happening in a separate PR.